marketplace/optimizely/src/AppPage/ContentTypes.js

```diff
@@ -136,7 +136,11 @@ export default function ContentTypes({
       <Paragraph className="f36-margin-top--m">
         Select the content types for which you want to enable A/B testing.
       </Paragraph>
-      <Button buttonType="muted" className="f36-margin-top--m" onClick={() => toggleModal(true)}>
+      <Button
+        buttonType="muted"
+        className="f36-margin-top--m"
+        onClick={() => toggleModal(true)}
+        data-testId="add-content">
```

Comment:
All Forma components have special `testId` properties for this purpose and you need to use them instead.

Reply:
I ran into a problem when using the `testId` property. For example my code would look like this:

```
testId={testId}
```

However, the rendered attribute would be this
```
data-test-id={testId}
```

And it appears that `getByTestId` isn't smart enough to pick up attributes like `data-test-id`. It seems to only look for something like `data-testid` or `data-testId`. 

Am I missing something here?

Reply:
you can change selector using `configure` => https://github.com/contentful/user_interface/blob/master/tools/testing/setup-tests.js#L14

Reply:
All set: ac628cf7aa27fc33628b43e897c70519de98cdd6